### PR TITLE
Fixes #2748: apoc.temporal.toZonedTemporal missing in temporal docs page

### LINKED ISSUE #2748: apoc.temporal.toZonedTemporal missing in temporal docs page
it is mentioned here

https://neo4j.com/labs/apoc/4.4/temporal/datetime-conversions/#datetime-available-procedures-apoc.date.parseAsZonedDateTime

 that it has been moved but doesn't show up here

https://neo4j.com/labs/apoc/4.4/temporal/temporal-conversions/

### PR DESCRIPTION
**Pull request by vga91**
Fixes #2748

Other docs fixes:
- image in `apoc.nodes.cycles.adoc`

### CODE REVIEW

File: docs/asciidoc/modules/ROOT/pages/temporal/datetime-conversions.adoc [hunk 1]
```diff
@@ -273,7 +273,7 @@ This function parses a date string in a specified format and converts it to a Ne
 
 [NOTE]
 ====
-This function has been renamed to `apoc.temporal.toZonedDateTime` and moved to the temporal section.
+This function has been renamed to `apoc.temporal.toZonedTemporal` and moved to the temporal section.
```
Comment (Lojjs) [inline docs/asciidoc/modules/ROOT/pages/temporal/datetime-conversions.adoc:276]: Maybe this could be rephrased so it is even clearer that`apoc.date.parseAsZonedDateTime` function is deprecated and `apoc.temporal.toZonedTemporal` is what should be used instead?

File: docs/asciidoc/modules/ROOT/partials/usage/apoc.nodes.cycles.adoc [hunk 2]
```diff
@@ -48,7 +48,7 @@ Furthermore, we can specify a `maxDepth` to consider cycles with only `n` interm
 MATCH (m1:Start) WITH collect(m1) as nodes CALL apoc.nodes.cycles(nodes, {maxDepth: 1}) YIELD path RETURN path
 ----
 
-image::cycles_max_depth_1.png
+image::cycles_max_depth_1.png[width=800]
```
Comment (Lojjs) [inline docs/asciidoc/modules/ROOT/partials/usage/apoc.nodes.cycles.adoc:51]: This change feels unrelated, was it intentional?

**Review by Lojjs — CHANGES_REQUESTED** [review]
The new docs looks good, just two minor comments.

File: docs/asciidoc/modules/ROOT/partials/usage/apoc.nodes.cycles.adoc (hunk 2 above)
Comment (vga91) [reply docs/asciidoc/modules/ROOT/partials/usage/apoc.nodes.cycles.adoc:51]: Yes, without the `[width=800]` (more precisely without square brackets), the image is not rendered.

File: docs/asciidoc/modules/ROOT/pages/temporal/datetime-conversions.adoc (hunk 1 above)
Comment (vga91) [reply docs/asciidoc/modules/ROOT/pages/temporal/datetime-conversions.adoc]: Changed

File: docs/asciidoc/modules/ROOT/pages/temporal/datetime-conversions.adoc [hunk 3]
```diff
@@ -273,7 +273,7 @@ This function parses a date string in a specified format and converts it to a Ne
 
 [NOTE]
 ====
-This function has been renamed to `apoc.temporal.toZonedDateTime` and moved to the temporal section.
+This function has been deprecated by `apoc.temporal.toZonedTemporal`, therefore it's recommended to use this one instead.
```
Comment (Lojjs) [inline docs/asciidoc/modules/ROOT/pages/temporal/datetime-conversions.adoc:276]:
```suggestion
This function has been deprecated by `apoc.temporal.toZonedTemporal`, therefore it's recommended to use that one instead.
```

Comment (Lojjs) [reply docs/asciidoc/modules/ROOT/pages/temporal/datetime-conversions.adoc:276]: I think this might be clearer?
